### PR TITLE
Set up seed data & deploying to Heroku

### PR DESCRIPTION
**Pull request by irisfaraway**
https://eaflood.atlassian.net/browse/WC-121

The app relies on both registrations and users being created by the existing service. We'll need some seed data for existing users and their registrations so we can test the app independently.

This PR will also cover any config changes required to deploy to Heroku for testing.

### CODE REVIEW

**Comment by irisfaraway** [comment]
Now deploying to https://waste-carriers-renewals.herokuapp.com/

File: app/models/conviction_search_result.rb [hunk 1]
```diff
@@ -5,12 +5,12 @@ class ConvictionSearchResult
...
   # TODO: Confirm types
-  field :match_result, type: String
-  field :matching_system, type: String
-  field :reference, type: String
-  field :matched_name, type: String
-  field :searched_at, type: DateTime
-  field :confirmed, type: Boolean
-  field :confirmed_at, type: DateTime
-  field :confirmed_by, type: String
+  field :matchResult,     type: String
```
Comment (Cruikshanks) [inline app/models/conviction_search_result.rb]: There are a number of changes to set field names to match what the current document looks like in the db. However we can keep our more ruby like syntax using field aliases

```ruby
field :matchingSystem, as: :matching_system, type: String
```

<https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#aliasing-fields>

Might help bring the code base to a more familiar ruby footing??

Comment (irisfaraway) [reply app/models/conviction_search_result.rb]: Good point. I'd definitely prefer more consistency with Ruby syntax so I'll make the change!

**Comment by irisfaraway** [comment]
@Cruikshanks I've swapped as much over to snake_case as possible if you want to have another look. (Massive PR now, sorry!)

**Pull request merged**